src/main/js/components/command-palette/index.js

```diff
@@ -32,6 +32,8 @@ function init() {
     searchResultsContainer,
     () => searchResults.querySelectorAll("a"),
     hoverClass,
+    () => {},
```

Comment:
would have been nicer if this was being passed an object so as I didn't have to pass this as well